src/create-glamorous.js

```diff
@@ -149,6 +149,13 @@ function createGlamorous(splitProps) {
         },
       })
 
+      function withComponent(newComp, options) {
+        return glamorous(newComp, {
+          forwardProps: GlamorousComponent.forwardProps,
+          ...options,
```

Comment:
![screen shot 2017-06-08 at 2 13 04 pm](https://user-images.githubusercontent.com/656630/26943745-9d886586-4c54-11e7-8926-a6f0d6fef5b4.png)

Whelp, apparently it doesn't. Makes sense I suppose:

![screen shot 2017-06-08 at 2 14 01 pm](https://user-images.githubusercontent.com/656630/26943777-bcd3f504-4c54-11e7-8f0d-2daae56471e9.png)